### PR TITLE
patches_nlp

### PR DESCRIPTION
**Pull request by ekmb**
- fix intent-slot dataset: replaced [cls] and [sep] to support other models
- added roberta and albert support to toekn_cl and punctuation model
- fixed args names for consistency

Signed-off-by: Evelina Bakhturina <ebakhturina@nvidia.com>

### CODE REVIEW

File: examples/nlp/token_classification/token_classification_infer.py [hunk 1]
```diff
@@ -50,15 +69,15 @@
     help="Whether to take predicted label in brackets or \
                     just append to word in the output",
 )
-parser.add_argument("--work_dir", default='output/checkpoints', type=str)
+parser.add_argument("--checkpoint_dir", default='output/checkpoints', type=str)
 parser.add_argument("--labels_dict", default='label_ids.csv', type=str)
 parser.add_argument("--amp_opt_level", default="O0", type=str, choices=["O0", "O1", "O2"])
```
Comment (VahidooX) [inline examples/nlp/token_classification/token_classification_infer.py]: Do we need amp_opt_level in inference files?

File: examples/nlp/token_classification/punctuation_capitalization_infer.py [hunk 2]
```diff
@@ -73,8 +92,8 @@
```
Comment (VahidooX) [inline examples/nlp/token_classification/punctuation_capitalization_infer.py]: Do we need amp_opt_level in inference files?

File: examples/nlp/token_classification/token_classification_infer.py [hunk 3]
```diff
@@ -50,15 +69,15 @@
...
 
 args = parser.parse_args()
 logging.info(args)
 
-if not os.path.exists(args.work_dir):
-    raise ValueError(f'Work directory not found at {args.work_dir}')
+if not os.path.exists(args.checkpoint_dir):
+    raise ValueError(f'Work directory not found at {args.checkpoint_dir}')
```
Comment (VahidooX) [inline examples/nlp/token_classification/token_classification_infer.py]: Checkpoint directory not found.

Comment (ekmb) [reply examples/nlp/token_classification/token_classification_infer.py]: updated the warning

File: examples/nlp/token_classification/punctuation_capitalization_infer.py (hunk 2 above)
Comment (ekmb) [reply examples/nlp/token_classification/punctuation_capitalization_infer.py]: removed

**Pull request merged**